README.md

```diff
@@ -24,7 +24,7 @@ Running with docker
 git clone git@github.com:pru-mike/rocketchat-jira-webhook.git
 cd rocketchat-jira-webhook
 docker build . -t rocketchat-jira-webhook
-cp example/minimal.toml config.toml
```

Comment:
Looks like `docker run` doesn't work like this anymore. Can you please fix readme 'Running with docker' accordingly to your modifications?

Reply:
Thanks, fixedthanks, fixed